docs/docs/middlewares/tracking.md

```diff
@@ -4,19 +4,28 @@ Sets tracking codes to template.
 ## Options
 option           | type   | required | description
 -----------------|--------|----------|------------
-google_analytics | string | ❌       | The Account id string
+google_analytics | string | ❌       | Path to the file that contains tracking code.
 
 **Example**
```

Comment:
Please add example of `js/google-analytics.js` top make it crystal clear how it is meant to work.